tests/integ/scala/test_update_delete_merge_suite.py

```diff
@@ -74,6 +74,18 @@ def test_update_rows_in_table(session):
     assert "condition should also be provided if source is provided" in str(ex_info)
 
 
+def test_update_rows_nondeterministic_update(session):
```

Comment:
@sfc-gh-aling I didn't include this in local test yet. It's related to session parameters. We might need to think about supporting some session parameters for local test in the future. Should we create a follow-up JIRA?